pkg/adapter/mtping/pingsource_test.go

```diff
@@ -70,19 +71,19 @@ func TestAllCases(t *testing.T) {
 			Name: "valid schedule",
 			Key:  pingsourceKey,
 			Objects: []runtime.Object{
-				NewPingSourceV1Beta1(pingSourceName, testNS,
```

Comment:
Can you add test for `contentType` and `dataBase64`

Reply:
done